### PR TITLE
Add missing role error page

### PR DESCRIPTION
**Pull request by yuxitang-amzn**
*Issue #, if available:*

*Description of changes:*
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/plugin.ts [hunk 1]
```diff
@@ -129,7 +146,7 @@ export class OpendistroSecurityPlugin
           ![APP_ID_DASHBOARDS, APP_ID_HOME, APP_ID_MULTITENANCY].includes(app.id)
         ) {
           return {
-            status: AppStatus.inaccessible,
+            status: AppStatus.accessible,
```
Comment (zengyan-amazon) [inline public/plugin.ts:149]: is this intended?

Comment (yuxitang-amzn) [reply public/plugin.ts]: Nope removed.

File: common/index.ts [hunk 2]
```diff
@@ -16,14 +16,25 @@
...
 export const CONFIGURATION_API_PREFIX = 'configuration';
 export const API_ENDPOINT_AUTHINFO = API_PREFIX + '/auth/authinfo';
-export const LOGIN_PAGE_URI = '/app/login';
+export const LOGIN_PAGE_URI = '/app/' + APP_ID_LOGIN;
 export const SELECT_TENANT_PAGE_URI = '/app/select_tenant';
+<<<<<<< HEAD
+export const CUSTOM_ERROR_PAGE_URI = '/app/' + APP_ID_CUSTOMERROR;
+=======
+export const CUSTOM_ERROR_PAGE_URI = '/app/customerror';
+>>>>>>> 7e2f731a0be938c5172808186a497bc7d2fcd204
```
Comment (zengyan-amazon) [inline common/index.ts:32]: looks like not resolved

Comment (yuxitang-amzn) [reply common/index.ts]: Resolved now

**Review by zengyan-amazon — APPROVED** [review]

**Pull request merged**